ops/conda_env/aarch64_test.yml

```
@@ -26,7 +26,7 @@ dependencies:
 - awscli
 - numba
 - llvmlite
-- loky
+- loky>=3.5.1
```

Instead of requiring latest, we could simply disallow the problematic version

This could also be done in `setup.py` and other environment files

```suggestion
- loky!=3.5.0
```

---

Thank you for the suggestion! I think I would like to keep it as the latest for the CI, it's only used by the tests and we don't want to deal with more than one version in the CI.